configs/dlstreamer/framework-pipelines/yolov5_pipeline/yolov5-gpu.env

```diff
@@ -0,0 +1,17 @@
+GST_PIPELINE_LAUNCH=/home/pipeline-server/framework-pipelines/yolov5_pipeline/yolov5s.sh
+OCR_DEVICE=
+LOG_LEVEL=ERROR
+GST_DEBUG=0
+LOW_POWER=
+CPU_ONLY=
+AUTO_SCALE_FLEX_140=
+DECODE=! vaapidecodebin
+DEVICE=GPU
+PRE_PROCESS=pre-process-backend=vaapi-surface-sharing pre-process-config=VAAPI_FAST_SCALE_LOAD_FACTOR=1
```

Comment:
the value of the env variable needs to be double quoted since there is space and equal sign in between.